.github/actions/setup-identity/action.yaml

```diff
@@ -16,9 +16,11 @@ outputs: {}
 runs:
   using: "composite"
   steps:
+    # add safe.directory is repeated here because it doesn't propagate over the checkout action
     - name: Setup git config
       shell: bash
       run: |
+        git config --global --add safe.directory '/__w/eksctl/eksctl'
```

Comment:
Without this, the release breaks. Turns out, it doesn't work ( no idea why it did during testing but now it doesn't lol ).

It's not propagating the setting. And `GITHUB_WORKSPACE` is not this folder so setting that is actually not the thing that fixed the thing apparently.

This is however so fingers crossed that nothing else will break here. I ran all action short on doing an actual release, so I'm hoping there won't be any more problems.

There are ample of issues open that concern this issue on checkout's GitHub page which I linked to in our issue, so tracking that it is sufficient. This is the suggested workaround.